test/factories.rb

```diff
@@ -0,0 +1,50 @@
+FactoryGirl.define do
+  factory :product do
+    name "My Amazing Article"
+    available true
+    on_hand_count 10
+    price 10.00
+  end
+
+  factory :user do
+    user_name 'John Doe'
+    email 'fake.email@gmail.com'
+    password 'password123'
+  end
+
+  factory :state do
+    abbreviation 'WI'
+    name 'Wisconsin'
+  end
+
+  factory :address do
+    street_addr '12b baker street'
+    association :state, factory: :state
```

Comment:
@austinkahly we don't but I can't recall the reasoning. Pretty sure they're the preferred method tho